lib/asciidoctor/inline_parser/inline_node.rb

```diff
@@ -0,0 +1,90 @@
+module Asciidoctor
+  module InlineParser
+    # Inline node
+    class Node
+      # The unparsed text that may contain formatting marks and inline elements
+      attr_accessor :source
+
+      # The parsed text attached to the current node
+      attr_accessor :text
+
+      # Node parent
+      attr_accessor :parent
+
+      # Children nodes
+      attr_accessor :children
+
+      def initialize
+        @children = []
+      end
+    end
+
+    class Strong < Node
```

Comment:
@mojavelinux If we want to ease migration with the existing converters, we should probably introduce a `Formatted` class (equivalent to `inline_quoted`)